website/docs/authenticating_via_msi.html.markdown

```diff
@@ -7,17 +7,17 @@ description: |-
 
 ---
 
-# Authenticating to Azure Resource Manager using Managed Service Identity
+# Azure Provider: Authenticating using Managed Service Identity
 
-Terraform supports authenticating to Azure through Managed Service Identity, Service Principal or the Azure CLI.
+Terraform supports authenticating to Azure through Managed Service Identity, a Service Principal, or the Azure CLI.
 
-Managed Service Identity can be used to access other Azure Services from within a Virtual Machine in Azure instead of specifying a Service Principal or Azure CLI credentials.
+Instead of specifying a Service Principal or Azure CLI credentials, Managed Service Identity can be used to access other Azure Services from within an Azure VM. 
 
 ## Configuring Managed Service Identity
 
-Managed Service Identity allows an Azure virtual machine to retrieve a token to access the Azure API without needing to pass in credentials. This works by creating a service principal in Azure Active Directory that is associated to a virtual machine. This service principal can then be granted permissions to Azure resources.
-There are various ways to configure managed service identity - see the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/active-directory/msi-overview) for details.
-You can then run Terraform from the MSI enabled virtual machine by setting the `use_msi` provider option to `true`.
+Managed Service Identity allows an Azure virtual machine to retrieve a token to access the Azure API without needing to pass in credentials. This is accomplished by creating a service principal in Azure Active Directory that is associated to a virtual machine. The resulting service principal can then be granted permissions to Azure resources.
```

Comment:
since SP's can be granted access to API's too - perhaps it's worth calling that out here?

```suggestion
Managed Service Identity allows an Azure virtual machine to retrieve a token to access the Azure API without needing to pass in credentials. This is accomplished by creating a service principal in Azure Active Directory that is associated to a virtual machine. The resulting service principal can then be granted permissions to Azure resources and API's.
```

Reply:
Corrected.

Reply:
@tombuildsstuff incorporated your feedback. Ready for your review.
(Adding @metacpp for his info.)